modules/order/src/Form/OrderForm.php

```diff
@@ -32,26 +38,44 @@ public function form(array $form, FormStateInterface $form_state) {
 
     $form['order_state'] = [
```

Comment:
Why are we keeping "order_state" as the key? Doesn't match the "Order information" label.